compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -202,23 +200,18 @@ impl<'tcx, Tag: Copy, Extra: AllocationExtra<Tag>> Allocation<Tag, Extra> {
     fn get_bytes_internal(
         &self,
         cx: &impl HasDataLayout,
-        ptr: Pointer<Tag>,
-        size: Size,
+        range: AllocRange,
```

Comment:
I went back-and-forth between passing around two `Size`s vs something more structured like `AllocRange`... sadly `Range` is not very suited since we want to pass offset and length, whereas a `Range` would mark beginning and end.